web/pgadmin/tools/erd/__init__.py

```diff
@@ -386,6 +386,20 @@ def register_preferences(self):
             )
         )
 
+        self.preference.register(
+            'options',
+            'table_relations_depth',
+            gettext('Table Relations Depth'),
```

Comment:
Depth of table relations or Depth of related tables?

Reply:
It is depth of `relations`. Depth of table has no meaning.